### ISSUE TITLE
LG magic remote, webos 5

### ISSUE THREAD

**Issue by damaskuss** [Simon34545/lginputhook#1]
Some buttons not disable. For example 1028 and 1021 (stb menu, stb power)

**Comment by Simon34545** [comment]
when you press these buttons, they show up in the logs, right?

**Comment by damaskuss** [comment]
yes it appears. I can assign another action to this button (STB menu), but the stb menu will still be enabled along with what I assigned, that is, as if the stb menu works at a higher level.
I'm sorry, I'm writing through a translator.

**Comment by Simon34545** [comment]
it might be possible something else is handling the buttons before they get processed by the input hook...
